src/price_oracle.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package helium;
+
+message price_oracle_resp_v1 {
```

Comment:
@madninja @bbalser lmk if the response for price oracles needs to contains something else to make sure its unique and verifiable?

Reply:
if I interpret hashc0de's conversation I believe we'll need a token name/type in there too.. and the request could either request a specific token or just request once and get all of them back.  I guess requesting a specific token type initially (to get like MOBILE -> USD)

Reply:
Should they be signed or do we expect trusted communication between the price oracle and the other oracles? If I followed the conversation correctly this service is only an abstraction for the other oracle infrastructure to query?

Reply:
I don't currently see a need to sign the response it's independently verifiable on the Solana chain if needed